_pages/en_US/installing-boot9strap-(hbl-usm).txt

```diff
@@ -0,0 +1,86 @@
+---
+title: "Installing boot9strap (HBL-USM)"
+---
+
+{% include toc title="Table of Contents" %}
+
+### Required Reading
+
+In order to exploit the SAFE_MODE firmware of our system, we need to inject an exploited WiFi profile.
+
+As we already have Homebrew access, we can use slotTool to do this.
+
+Once the WiFi profile has been injected, we will use SAFE_MODE, which is a recovery feature present on all 3DS consoles, to activate the exploited WiFi profile.
+
+These instructions work on USA, Europe, Japan, and Korea region consoles as indicated by the letters U, E, J, or K after the system version.
+
+If your (Right/Left Shoulder), (D-Pad Up) or (A) buttons do not work, you will need to use a [Legacy Method](legacy-methods). For assistance with this matter, join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp) and ask, in English, for help.
+{: .notice--danger}
+
+### What You Need
+
+* The latest release archive for [Luma3DS](https://github.com/AuroraWright/Luma3DS/releases/latest)
+* The latest release archive for [unSAFE_MODE](https://github.com/zoogie/unSAFE_MODE/releases/latest)
+
+#### Section I - Prep Work
+
+1. Copy `boot.firm` and `boot.3dsx` from the Luma3DS `.zip` to the root of your SD card, replacing any existing files
+  + The root of the SD card refers to the initial directory on your SD card where you can see the Nintendo 3DS folder, but are not inside of it
+1. Create a folder named 3ds on the root of your SD card if it does not already exist
+1. From the slotTool folder from the unSAFE_MODE `.zip`, copy `slotTool.3dsx` to the `3ds` folder
```

Comment:
A bit oddly worded, but might be unnecessary if slotTool is added to the new-browserhax archive

Reply:
pending changes from zoogie